lib/Extractor/Solver.cpp

```diff
@@ -111,6 +115,13 @@ class BaseSolver : public Solver {
       }
     }
 
+    if (InferInsts && SMTSolver->supportsModels()) {
+      InstSynthesis IS;
+      EC = IS.synthesize(SMTSolver.get(), BPCs, PCs, LHS, RHS, IC, Timeout);
+      if (EC || RHS)
+        return EC;
+    }
+
     RHS = 0;
```

Comment:
Yes, but we still have to cover the path where synthesis is not invoked at all, right?